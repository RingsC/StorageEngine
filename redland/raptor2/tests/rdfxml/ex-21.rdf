<?xml version="1.0"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
	 xmlns:rd="http://example.org/foo#"
	 xmlns:r="http://example.org/bar#">
  <!-- Tests that the rdf prefix is recognised correctly -->
  <rdf:Description rdf:about="http://example.org/thing1" rd:type="val1" r:type="val2" />
  <rd:Description rdf:about="http://example.org/thing2" r:prop1="val1" />
  <r:Description r:about="http://example.org/thing3" r:prop1="val1" />
</rdf:RDF>
