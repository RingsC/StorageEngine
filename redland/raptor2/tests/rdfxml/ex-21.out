<http://example.org/thing1> <http://example.org/foo#type> "val1" .
<http://example.org/thing1> <http://example.org/bar#type> "val2" .
<http://example.org/thing2> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/foo#Description> .
<http://example.org/thing2> <http://example.org/bar#prop1> "val1" .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://example.org/bar#Description> .
_:genid1 <http://example.org/bar#about> "http://example.org/thing3" .
_:genid1 <http://example.org/bar#prop1> "val1" .
